### ISSUE TITLE
Is this document consistent with IOOS SOS Milestone 1??

### ISSUE THREAD

**Issue by emiliom** [ioos/conventions-for-observing-asset-identifiers-hugo#3]
Based on a _cursory_ scan, this document looks like it's largely unchanged from the original, old IOOS asset identifier document. But that document did not reflect the decisions made for IOOS SOS Milestone 1 back in 2012. See these documents and list threads:
https://code.google.com/p/ioostech/wiki/OutdatedSOSguidelines#Asset_URN_identifiers
https://code.google.com/p/ioostech/issues/detail?id=14
https://groups.google.com/forum/#!topic/ioostech_dev/F1SBuamGUrI/discussion
The first link is a succinct but accurate summary of those decisions. That's what's implemented in the SOS servers (52N for sure, and presumably ncSOS as well).

**If** my cursory scan is in fact correct (I could've missed something!), it would be good to make this, umm, "legacy" (sort of) document consistent with the IOOS SOS M1 decisions & conventions. I'm personally rather fond of [the brief IOOS SOS M1 description on this topic; then again, I'm biased, b/c I wrote the bulk of it ;)](https://code.google.com/p/ioostech/wiki/OutdatedSOSguidelines#Asset_URN_identifiers). One of our findings during that 2012 discussion was that the old Geo-Ide asset identifier document had several internal inconsistencies and concepts that were not fully fleshed out and were misleading. Though maybe the edits listed as "0.1 Updated draft  2013-12-06" in the revision history already addressed some of those issues?

As @dpsnowden and @abirger will recall, our IOOS SOS M1 discussions on this topic involved lots of sweat and blood, but they did result in a tight and consistent set of decisions and conventions. Though it may well be time to revisit them ...

**Comment by dpsnowden** [comment]
Yes, the document is unchanged.  That was deliberate and just a first step.  My intention is to publish the doc as it was in it's previous state, exactly.  We'll create a release and version number to match whatever convention was in place originally.  I can't remember if it was 0.9 or 1.0.  

From that point we will, in good github style, edit the doc through pull requests etc.  Once we stabilize the next version we will release it and increment the version number appropriately.  The next version number will be chosen based on how significant the changes are and we'll try to follow semver.org principles as we're trying to do for software.  It's a learning process and we're just getting started implementing versioning widely.

Thanks for summarizing some of the issues to be resolved and included in the next version.  We aim to have all of our docs converted to markdown, rendered into html and published with updates by the end of the calendar year.  When we get to the editing stage I'm sure we'll have plenty of questions/clarifications that we'd like to hear straight from the horses mouth.

**Comment by emiliom** [comment]
Thanks for that description of the goals and process. That makes sense. I'm happy to help when the time comes.

**Comment by abirger** [comment]
@emiliom , I've made a "draft" branch in my fork of this IOOS repository at https://github.com/abirger/conventions-for-observing-asset-identifiers/tree/drafts/doc , and keep there a somewhat updated version of the initial document. The update includes the changes that you are talking about, as well as some other modifications. This updated version should be periodically pulled back to the Draft-Branch of the IOOS official repository, so interested parties can see it but it stays somewhat hidden. 
The idea behind that "draft" branch was to keep the process of updating and modifying of the original asset identifier document kind of under the carpet until the updated copy is ready to replace the original. I am not sure that it was the right idea but I could not think of anything better at the moment.

**Comment by emiliom** [comment]
No worries, @abirger . The only reason I got to that document is b/c I was looking around to see if there was already a replacement on github for https://code.google.com/p/ioostech/wiki/OutdatedSOSguidelines#Asset_URN_identifiers. I wanted to avoid citing that deprecated ioostech document in an ioos_tech list email (that I sent yesterday), if at all possible.

**Comment by srstsavage** [comment]
Found this after spotting some inconsistencies with the IOOS URN spec and our SOS usage. Should we be updating the [draft 0.5 spec](https://github.com/ioos/conventions-for-observing-asset-identifiers/blob/master/doc/ioos_assets_wiki_upd_v0_5_draft.md) with consistency fixes? I hope so...

**Comment by dpsnowden** [comment]
Yeah, I think that we need to update this doc after the RA meeting.  @kwilcox will tell us all about the modifications he's proposing.

**Comment by srstsavage** [comment]
Ok, great. I made a pull request for some formatting improvements to the current 0.5 draft in the meantime:

https://github.com/ioos/conventions-for-observing-asset-identifiers/pull/5
